tests/event_discovery.tests/features.json

```diff
@@ -7,12 +7,25 @@
   "timestamp": "2018-08-26T21:42:12.237Z",
   "generation": "2018-08-26T21:37:12Z",
   "features": {
-    "system": {
-      "baseline": {
-        "last_config": {}
+    "endpoint": {
+      "features": {
+        "config": {
+          "stage": "alpha"
```

Comment:
Is this the stage of the device's implementation of this feature? In the case that all the tests under endpoint.config were stable, and/or endpoint.config itself was "stable" (if we assign stages to buckets), a device can still be "alpha"?